### ISSUE TITLE
New unit request: Atoms per Litre

### ISSUE THREAD

**Issue by dr-shorthair** [qudt/qudt-public-repo#220]
http://registry.it.csiro.au/def/environment/unit/AtomsPerLitre

http://registry.it.csiro.au/def/environment/unit is a list of units that were needed but not available in the QUDT catalogue. 
I've marked the ones that were subsequently added to QUDT `superseded` with pointers to the items in QUDT. 
Now that QUDT has a means to accept external requests, we should be able to move the missing items into QUDT and retire the CSIRO register.

**Comment by steveraysteveray** [comment]
I note that the ones marked "superseded" usually identify several qudt Unit subClasses.

I think we discussed this earlier, where we deprecated all the various subClasses of qudt:Unit, because many units were belonging in many of the subClasses. Instead, we're thinking of using the vocabulary of qudt:Discipline instances, organized as a skos hierarchy. None of them have been scrubbed yet, and there's still the open question of whether a given Discipline instance should point to relevant quantity kinds and applicable units, or just quantity kinds. Also, should a given specialized Discipline lexicon be made up of the quantity kinds directly referenced, or also those referenced by any skos:Broader discipline?

These are design questions that need resolving.

**Comment by dr-shorthair** [comment]
Yeah - we were a bit exuberant in adding all those types. This work was done about 6 years ago. We would expect to re-formulate and not carry all those types over.

**Comment by Toby-Broom** [comment]
This already exists, https://qudt.org/vocab/unit/NUM-PER-L as per IUPAC 2.10 in green book the number of entities (e.g. molecules, atoms, ions, formula units) has no units as per the qk:NumberDensity this applies to atoms and so on.

My only comment would be that maybe NumberOfParticles could be refactored to be number of entities since it looks to be the same as the IUPAC for the number of entities, where a particle is also an entity.

@fkleedorfer
